docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -0,0 +1,575 @@
+---
+title: Internationalization (i18n) - Strapi Developer Documentation
+description: Instructions on how to use Strapi Content API with the Internationalization (i18n) optional plugin
+---
+
+# Internationalization (i18n)
+<!-- ? wondering if it would be a good idea to add the 🌍 icon to the h1 — if yes, we should do this for the h1 of other plugins as well -->
+
+> Internationalization is the design and development of a product, application or document content that enables easy localization for target audiences that vary in culture, region, or language.
+>
+> -- <cite>[W3C](https://www.w3.org/International/questions/qa-i18n.en#i18n)</cite>
```

Comment:
I'm not very keen on that idea of using a quote 🤔  I think it might be interesting to link a full definition of i18n in our documentation, but in my opinion something like "For more information about the concept of internationalization, please refer to [...]" would be enough.